blog/_posts/2022-03-30-tilt-news-march-2022.md

```diff
@@ -0,0 +1,56 @@
+---
+slug: "tilt-news-march-2022"
+date: 2022-03-30
+author: lian
+layout: blog
+title: "Tilt News, March 2022"
+image: "/assets/images/tilt-news-march-2022/pexels-green-windmills.jpg"
+image_caption: "A dutch windmill and house all in green. Photo by Helena Jankovičová Kováčová from <a href='https://www.pexels.com/photo/a-green-windmill-near-the-house-6470648/'>Pexels</a>"
+description: "Updated posts and cool new stuff we found during spring-cleaning 🌼"
+tags:
+  - news
+---
+
+你好 Tilt 用户,
+
+This month, vacation season started at Tilt, so we’re going to have a very brief update.
+
+## Updates from the ecosystem
+Developer tooling is gaining more and more attention from engineers and investors alike. While that means tougher competition, in our opinion, this will also improve the quality of the overall ecosystem.  
+So, we’re excited to congratulate [Okteto on their Series A](https://www.okteto.com/blog/a-15m-series-a-to-disrupt-modern-development/) and [Signadot](https://www.prnewswire.com/news-releases/y-combinator-alum-signadot-raises-4m-led-by-redpoint-ventures-to-reimagine-microservices-testing-on-kubernetes-301488164.html) on their Seed funding!
+
+## Optimize your Tilt experience
```

Comment:
"Optimize your Tilt experience" is kind of hand-wavey and formal. how about - "Our favorite changes if you upgraded Tilt this month". or maybe that's too informal :stuck_out_tongue:

Reply:
I'll change it to 
**If you upgraded Tilt this month, you’ll have more fun...**